frontend/src/environments/environment.ts

```diff
@@ -7,6 +7,7 @@ export const environment = {
     coreUrl: 'http://localhost:5050',
     notifierUrl: 'http://localhost:5070',
     consoleUrl: 'http://localhost:44567',
+    sqlServiceUrl: 'https://localhost:7244',
```

Comment:
Can we make it via webapi?
FE sends request to webapi
WebAPI sends request to sqlService

Reply:
done